hack/jenkins/job-configs/global.yaml

```diff
@@ -11,9 +11,21 @@
 - publisher:
     name: gcs-uploader
     publishers:
-        # Use our script for build artifacts, since it's more flexible.
         - postbuildscript:
             builders:
+                # Dump master and node logs before uploading.
```

Comment:
This probably won't work here, actually, since it might break in non-e2e suites.